File: qc-core/src/test/java/com/eager/questioncloud/utils/Fixture.java
package com.eager.questioncloud.utils;

import com.navercorp.fixturemonkey.FixtureMonkey;
import com.navercorp.fixturemonkey.api.plugin.SimpleValueJqwikPlugin;
import com.navercorp.fixturemonkey.kotlin.KotlinPlugin;

public class Fixture {
    public static SimpleValueJqwikPlugin simpleValueJqwikPlugin = new SimpleValueJqwikPlugin().minStringLength(10).maxStringLength(15);
    public static FixtureMonkey fixtureMonkey = FixtureMonkey
        .builder()
        .plugin(simpleValueJqwikPlugin)
        .plugin(new KotlinPlugin())
        .defaultNotNull(true)
        .build();
}

File: qc-application/src/test/java/com/eager/questioncloud/application/utils/Fixture.kt
package com.eager.questioncloud.application.utils

import com.navercorp.fixturemonkey.FixtureMonkey
import com.navercorp.fixturemonkey.api.plugin.SimpleValueJqwikPlugin
import com.navercorp.fixturemonkey.kotlin.KotlinPlugin

object Fixture {
    private var simpleValueJqwikPlugin: SimpleValueJqwikPlugin =
        SimpleValueJqwikPlugin().minStringLength(10).maxStringLength(15)
    var fixtureMonkey: FixtureMonkey = FixtureMonkey
        .builder()
        .plugin(simpleValueJqwikPlugin)
        .plugin(KotlinPlugin())
        .defaultNotNull(true)
        .build()
}